### PR TITLE
INS-2994 Refactor Import process

### PR DESCRIPTION
**Pull request by Zemidar**

### CODE REVIEW

File: sell/sell-api.yaml [hunk 1]
```diff
@@ -2054,17 +2033,15 @@ paths:
...
-          multipart/form-data:
-            schema:
-              type: object
-              properties:
-                mappings:
-                  type: string
-                file:
-                  type: string
-                  format: binary
+      parameters:
```
Comment (olyavertwist) [inline sell/sell-api.yaml:2036]: parameters:
        - name: mappings
          description: This parameter defines mapping between incoming data and Inperium Sell properties.
          in: query
          schema:
            type: string
        - name: linkToFile
          description: The link to the import file.
          in: query
          schema:
            type: string

**Review by olyavertwist — COMMENTED** [review]
I left a comment with descriptions

File: sell/sell-api.yaml [hunk 2]
```diff
@@ -152,24 +149,6 @@ tags:
       Configure integration with Xero.
 
 paths:
-  /health:
```
Comment (aehrenthal) [inline sell/sell-api.yaml:155]: Is the health endpoint no longer required? I think we use it to ping the server on a regular basis to ensure uptime.

File: sell/sell-api.yaml [hunk 3]
```diff
@@ -2054,17 +2033,17 @@ paths:
...
-                file:
-                  type: string
-                  format: binary
+      parameters:
+        - name: mappings
+          description: This parameter defines mapping between incoming data and Inperium Sell properties.
+          in: query
+          schema:
+            type: string
+        - name: linkToFile
```
Comment (aehrenthal) [inline sell/sell-api.yaml:2042]:
```suggestion
        - name: fileUrl
```
I think we used "url" across the API to specify links. Just to be consistent.

**Review by aehrenthal — CHANGES_REQUESTED** [review]

File: sell/sell-api.yaml (hunk 2 above)
Comment (Zemidar) [reply sell/sell-api.yaml:155]: Actually Spring Boot has own endpoint for this purpose - actuator/health
Who is responsible for ping server? I'll send him new endpoint :)

File: sell/sell-api.yaml (hunk 3 above)
Comment (Zemidar) [reply sell/sell-api.yaml]: Fixed

**Review by RockMother — APPROVED** [review]

**Review by aehrenthal — APPROVED** [review]

File: sell/sell-api.yaml (hunk 2 above)
Comment (aehrenthal) [reply sell/sell-api.yaml:155]: Please ask @tsyrenov to change it in the AWS configs

**Pull request merged**